### ISSUE TITLE
Installation stopped working on Ubuntu 20.04

### ISSUE THREAD

**Issue by init-skr** [blue-yonder/turbodbc#357]
# Description

Hi everyone, was trying to upgrade a custom Docker image I use to install turbodbc and some other deps. It stopped working a couple of weeks ago. Did not change anything. Everything is pinned (except for boost install / stuff that is installed via apt) in python.

OS: Ubuntu 20.04
Python: 3.9
Turbodbc Version: 4.4.0 (also tried 4.4.3)

# Problem

The exact issue seems to be:

```bash
      /usr/bin/ld: cannot find -lturbodbc.cpython-39-x86_64-linux-gnu
      collect2: error: ld returned 1 exit status
      error: command '/usr/bin/g++' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for turbodbc
  Running setup.py clean for turbodbc
Failed to build turbodbc
Installing collected packages: turbodbc
  Running setup.py install for turbodbc: started
  Running setup.py install for turbodbc: still running...
  Running setup.py install for turbodbc: still running...
  Running setup.py install for turbodbc: finished with status 'error'
  error: subprocess-exited-with-error

[...]

      /usr/bin/ld: cannot find -lturbodbc.cpython-39-x86_64-linux-gnu
      collect2: error: ld returned 1 exit status
      error: command '/usr/bin/g++' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure
```

**Comment by david-engelmann** [comment]
@init-skr I recently had a similar install issue and documented how i was able to resolve it in [this comment](https://github.com/blue-yonder/turbodbc/issues/351#issuecomment-1122946938). I hope this helps you!

**Comment by init-skr** [comment]
Thanks @david-engelmann, did not have time yet but will check out your solution. 👍

**Comment by david-engelmann** [comment]
@init-skr In another comment in that issue I linked I wrote down what I thought the big takeaways were!

**Comment by Nicoretti** [comment]
Hi @init-skr 

you may want to reevaluate if your issue is fixed with `4.5.4`, the following fix https://github.com/blue-yonder/turbodbc/pull/359 may also has addressed your issue.

**Comment by init-skr** [comment]
Still didn't have time to evaluate @david-engelmann  advice(thanks again). 

@Nicoretti Thanks for the update hopefully I find time to check it soon. I'll close the issue for now and will reopen if it persists.
